polars/polars-time/src/windows/window.rs

```diff
@@ -218,9 +230,11 @@ impl BoundsIter {
                     let dt = dt.naive_utc();
                     let start = to(dt);
                     // apply the 'offset'
-                    let start = offset(&window.offset, start);
+                    // TODO remove unwrap once time zone is respected
+                    let start = offset(&window.offset, start, None::<&FixedOffset>).unwrap();
                     // and compute the end of the window defined by the 'period'
-                    let stop = offset(&window.period, start);
+                    // TODO remove unwrap once time zone is respected
+                    let stop = offset(&window.period, start, None::<&FixedOffset>).unwrap();
```

Comment:
I think we should add a

```rust
pub const NO_TIMEZONE: Option<&FixedOffset> = None;
```

in `polars-time/src/lib.rs` to make this easier to write.